solidity/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/keep-network/keep-ecdsa",
   "dependencies": {
-    "@keep-network/keep-core": ">1.2.4-rc <1.2.4",
+    "@keep-network/keep-core": ">1.3.0-pre <1.3.0",
```

Comment:
`>1.3.0-pre <1.3.0-rc` perhaps?

Reply:
we don't want to include release candidates?

Reply:
`-pre` should refer to `-pre`s. We should also update the version of this package on the top of the file.

Reply:
done